frontend/src/js/components/devices/DeviceListItem.tsx

```diff
@@ -77,17 +76,4 @@ const DeviceListItem = ({ columnHeaders, listItem: device, listState: deviceList
   );
 };
 
-const areEqual = (prevProps, nextProps) => {
-  if (
-    prevProps.idAttribute.attribute != nextProps.idAttribute.attribute ||
-    prevProps.idAttribute.scope != nextProps.idAttribute.scope ||
-    prevProps.selected != nextProps.selected ||
-    !deepCompare(prevProps.columnHeaders, nextProps.columnHeaders) ||
-    !deepCompare(prevProps.device, nextProps.device)
-  ) {
-    return false;
-  }
-  return deepCompare(prevProps.deviceListState, nextProps.deviceListState);
-};
-
-export default memo(DeviceListItem, areEqual);
```

Comment:
This was required for fuller device lists to allow resizing the columns smoothly back when the customizable columns were introduced. I haven't checked it locally, but please make sure this doesn't cause the resizing to become janky on slower machines. However this was >3 years ago, so more things might have changed...